src/test/resources/features/apiexamples/POST_Board.feature

```diff
@@ -1,7 +1,7 @@
 Feature: Board Controller
 
   Background: Set the authentication
-    Given I set authentication using "user1"
+    Given I set the authentication using "user1"
```

Comment:
this step is not defined (at least not in the same way) so it won't execute the test